nifi-assembly/pom.xml

```diff
@@ -19,6 +19,10 @@ language governing permissions and limitations under the License. -->
     <artifactId>nifi-assembly</artifactId>
     <packaging>pom</packaging>
     <description>This is the assembly Apache NiFi</description>
+    <!--Add a blank run.as in case RPM isn't being used -->
+    <properties>
+        <nifi.run.as />
```

Comment:
@trixpan Including this at the top level means that with the rpm profile, the bootstrap.conf included in the .tar.gz and .zip packages has run.as=nifi. Is this the right behavior?